.github/workflows/docker-ci.yml

```
@@ -2,155 +2,113 @@ name: Docker CI
+        uses: actions/upload-artifact@v4
+        with:
+          name: docker-artifact
+          path: /tmp/image.tar
 
-  notify:
+  # --- CALL DOWNSTREAM TEST ---
+  test:
     needs: build
-    if: github.event_name == 'pull_request' && needs.build.result == 'success'
-
+    uses: ./.github/workflows/ci.yaml
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/ci.yaml": open /home/jailuser/git/.github/workflows/ci.yaml: no such file or directory

(workflow-call)